<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>196</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>413</width>
    <height>196</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>413</width>
    <height>196</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Select Attribute</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="GBOkButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="GBExitButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="GBSelectAttribute">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: center;
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: black;
</string>
   </property>
   <property name="text">
    <string>Select an Atribute</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: center;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: black;
</string>
   </property>
   <property name="text">
    <string>Choose an attribute for grouping:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
